setup.py

```diff
@@ -309,6 +309,7 @@ def finalize_options(self):
         'scipy >= 0.18.1',
         'six >= 1.5.0',
         'smart_open >= 1.2.1',
+        'python-Levenshtein >= 0.10.2'
```

Comment:
This dependency might be temporary, since the current Levenshtein distance implementation is inefficient. We discussed more efficient bulk Levenshtein distance implementation in #1955.